### DISCUSSION TITLE
Ask about #simplex

### DISCUSSION THREAD

**Discussion by Kowersky** [IntellectualSites/FastAsyncWorldEdit#2224] [Q&A]
Hi. I want to make wall with #simplex but when i want make wall i have error
![image](https://github.com/IntellectualSites/FastAsyncWorldEdit/assets/43541615/76d3ddff-33d8-4e22-b284-f3ffe39fbc04)

**Comment by Kowersky** [comment]
![image](https://github.com/IntellectualSites/FastAsyncWorldEdit/assets/43541615/fbe70bf5-26b1-422f-a28f-7199250d0d3f)
![image](https://github.com/IntellectualSites/FastAsyncWorldEdit/assets/43541615/fab14452-f825-4418-8110-73364cd40392)
![image](https://github.com/IntellectualSites/FastAsyncWorldEdit/assets/43541615/41ed87ba-4498-4c88-afeb-adebd3840045)

**Comment by 360shvit** [comment]
FAWE is a direct replacement to Worldedit so you don't have to install it as a separate plugin. Also you may want to update to the latest build of Fawe. Prupur is also not recommended iirc so if you do not depend on it you may want to change to Paper. Providing a /fawe debugpaste may help the Fawe Team solving your the problem if the steps above do not solve it.

**Reply by Kowersky** [reply]
okay

**Reply by Kowersky** [reply]
i deleted worldedit but now i cant set blocks by command //set

**Reply by Kowersky** [reply]
> i deleted worldedit but now i cant set blocks by command //set

and i cant make any fawe command

**Reply by Kowersky** [reply]
maybe i must use paper
